src/libs/adc/adc.h

```diff
@@ -0,0 +1,105 @@
+/* Author: William Miles
```

Comment:
please use `/**` and doxygen style if possible

please include the license header